tests/shared/crclient/client.go

```diff
@@ -124,12 +142,22 @@ func (f Client) listPodList(list *corev1.PodList) error {
 	return nil
 }
 
-func (f Client) listNodeList(list *corev1.NodeList) error {
+func (f Client) listNodeList(list *corev1.NodeList, label string) error {
+	_, log := logger.GetNewContextWithLogger("0")
```

Comment:
You should probably just use `GetLogger`. Or don't log at all in this utility function.

Reply:
I'll remove the logging here.